### PR TITLE
utils/netutils: handle Atoi conversion error

### PR DESCRIPTION
**Pull request by unclejack**
The unit tests passed. The system tests are still running.

### CODE REVIEW

File: utils/netutils/netutils.go [hunk 1]
```diff
@@ -221,8 +221,8 @@ func ParseCIDR(cidrStr string) (string, uint, error) {
 	}
 
 	subnetStr := strs[0]
-	subnetLen, _ := strconv.Atoi(strs[1])
-	if subnetLen > 32 {
+	subnetLen, err := strconv.Atoi(strs[1])
+	if subnetLen > 32 || err != nil {
```
Comment (erikh) [inline utils/netutils/netutils.go]: use ParseInt instead.

Comment (erikh) [inline utils/netutils/netutils.go]: Seems atoi() is nothing like C atoi() so it's not broken. Sorry to confuse you.

Comment (unclejack) [inline utils/netutils/netutils.go]: No worries, it's all good.

**Comment by jainvipin** [comment]
LGTM

**Comment by erikh** [comment]
LGTM

**Pull request merged**